### PR TITLE
 #171186789 Fix ESLint and HoundCI Error Messages

### PR DESCRIPTION
**Pull request by Moise1**
#### What does this PR do?
- This PR enhances good code quality as required by ESLint and HoundCI

#### Description of Task to be completed?
- Setting the rules inside the eslintrc.json file correctly. 
- Deleting obsolete eslintrc.js

#### How should this be manually tested?

- HoundCI should not throw *no-def* and *'import' and 'export' may appear only with 'sourceType: module'*  errors

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#171186789](https://www.pivotaltracker.com/n/projects/2431452)

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

**Review by T2Wil — CHANGES_REQUESTED** [review]
Hey @Moise1 ,
some rules as described from PR descriptions are missing in the eslintrc.json file, you may consider adding them all.
`
"rules": {
    "one-var": 0,
    "no-undef": 0,
    "one-var-declaration-per-line": 0,
    "new-cap": 0,
    "consistent-return": 0,
    "no-param-reassign": 0,
    "comma-dangle": 0,
    "curly": [
      "error",
      "multi-line"
    ] 
`

**Comment by Moise1** [comment]
@T2Wil , which rules do you say are missing?

**Review by T2Wil — APPROVED** [review]

**Comment by T2Wil** [comment]
Ahh I am sorry @Moise1 , I had to click on the bottom left button to see the whole file content.
nice work !

**Review by Niyonsengaeric — APPROVED** [review]

File: .eslintrc.json [hunk 1]
```diff
@@ -6,8 +6,15 @@
     "es6": true,
     "mocha": true
   },
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
```
Comment (Kagorora) [inline .eslintrc.json:11]: @Moise1 help me understand what does this line do?

Comment (Moise1) [reply .eslintrc.json:11]: @Kagorora , It handles the below error message from HoundCI by specifying the ES version and sourceType.

 'import' and 'export' may appear only with 'sourceType: module' errors.

Comment (Kagorora) [reply .eslintrc.json:11]: @Moise1  I thought that what line 6 ` "es6": true,` was doing, however I more interested in  the functionality of this  `"sourceType": "module",`

**Pull request merged**